hal/src/photon/make_combined.sh

```diff
@@ -1,4 +1,23 @@
+#!/bin/bash
+
+## Building the combined Image
+# set WICED_SDK to point to the directory containing the photon-wiced repo, with the
+# `feature/combined-fw` branch checked out
+# (optional) set DEVICE_OS_DIR to point to the directory containing the device-os repo with the
+# necessary release tag checked out
+# Run this script, specifying the target PLATFORM_ID, e.g.: ./make_combined.sh PLATFORM_ID=6
+# This will build the artefacts to $(DEVICE_OS_DIR)/build/releases/release-$(DEVICE_OS_VERSION)-p$(PLATFORM_ID)
+# e.g. $(DEVICE_OS_DIR)/build/releases/release-1.2.1-p6 for Photon
+
 # ensure bytes are treated as bytes by tr or /xFF is output as a UTF-8 byte pair
 export LC_CTYPE=C
 
-gnumake -f wiced_test_mfg_combined.mk FIRMWARE=/spark/firmware WICED_SDK=/spark/photon-wiced  $*
+export PWD=$(cd $(dirname $BASH_SOURCE) && pwd)
+
+DEVICE_OS_DIR="${DEVICE_OS_DIR:-${PWD}/../../../}"
+if [ -z "${WICED_SDK}" ]; then
+    echo "WICED_SDK needs to be set"
+    exit 1
+fi
+
+make -f wiced_test_mfg_combined.mk FIRMWARE="${DEVICE_OS_DIR}" WICED_SDK="${WICED_SDK}" $*
```

Comment:
With all of the errors I encountered, I wished that when this finally completed it printed something in terminal to indicate it completed successfully since it does a lot of output prior to that.  Maybe something like "\r\n\r\nComplete!"

Reply:
It would also be good to add the full steps to test to the header of `make_combined.sh`

Reply:
> I found that before building the platform with build $ time ./make_release.sh --platform=photon --publish=1.2.1 I was getting this error:

That's probably a side-effect of not running a `clean` target when switching branches.